ebos/eclbasevanguard.hh

```diff
@@ -629,6 +630,7 @@ private:
     Opm::EclipseState* eclState_;
     Opm::Schedule* eclSchedule_;
     Opm::SummaryConfig* eclSummaryConfig_;
+    Opm::Python python;
```

Comment:
Is it ok to construct Python twice? I was not able to understand the table in Python.hpp... (Also, in that file, I think the doc has a typo in the example on line 53, I think "with" should be "without".)

Assuming constructing twice is fine, this PR is fine as a whole.

Reply:
> Assuming constructing twice is fine, this PR is fine as a whole.

It is safe to do so - but the best is to only construct one instance. The best solution would in my opinion be to get rid of the Internal variables in `eclbasevanguard`- in a fist of hubris I started on that in #2500 - but I don't know if life is long enough to follow that path.

Cool if you comment on the table in the upstream PR - if that was not useful it should be updated.

Reply:
> It is safe to do so - but the best is to only construct one instance. The best solution would in my opinion be to get rid of the Internal variables in `eclbasevanguard`- in a fist of hubris I started on that in #2500 - but I don't know if life is long enough to follow that path.

I agree. Also with the "maybe not worth it given average life expectancy" part...



> Cool if you comment on the table in the upstream PR - if that was not useful it should be updated.

I added a comment about what confused me.